File: E1/Task1/plot_function.m
% plot the powerspectrum
% Created by Martin Gren 2014-10-25.

% load the data file
data1 = importdata('function_phi0_f2.dat');
data2 = importdata('function_phiPiHalf_f1.dat');
data3 = importdata('function_phi0_f1.dat');
%plot
figure;
hold on;
plot(data1(:,1),data1(:,2),'-');
plot(data2(:,1),data2(:,2),'-.');
plot(data3(:,1),data3(:,2),'-.');
legend('\phi = 0, f = 2', '\phi = \pi/2, f = 1', '\phi = 0, f = 1');

% labels
xlabel('$t$','Interpreter','LaTeX');
ylabel('$h(t)$','Interpreter','LaTeX');

File: E1/Task1/plot_function.py
# plot the h(t)
# Created by Martin Gren 2014-10-25.

# imports
import matplotlib.pyplot as plt
import numpy as np

# input file
filename = 'function.dat'

# import data
data = np.loadtxt(filename)

# initial size of plot window
plt.figure(figsize=(8,6))

# plot
plt.plot(data[:,0], data[:,1],'-')

# labels
plt.xlabel('t / [arb. unit]', fontsize=20)
plt.ylabel('h(t) / [arb. unit]', fontsize=20)

# set tick fontsize
plt.yticks(fontsize=12)
plt.xticks(fontsize=12)

# display the plot

plt.savefig('h(t).pdf')
plt.show()
